builds/delete-serverless-service-build-pack/Jenkinsfile

```diff
@@ -258,7 +258,7 @@ node {
 				stage('Undeploy Service') {
 					for (_envId in environmentIds) {
 						current_environment_id = _envId
-						loadServerlessYml(service_config, _envId)
+            def has_serverless_yml = 	loadServerlessYml(service_config, _envId)
```

Comment:
what is the variable tracking?

Reply:
We are capturing the serverless yml preparation status in that  particular variable and if it is failing, we are going for aws cli cmd to remove the CF stack